ucm2/Intel/sof-hda-dsp/sof-hda-dsp.conf

```diff
@@ -104,15 +104,20 @@ If.dmic {
 		Type String
 		Empty "${var:DeviceDmic}"
 	}
-	False.If.Dmic0 {
-		Condition {
-			Type ControlExists
-			Control "name='Dmic0 Capture Volume'"
+	False {
+		If.Dmic0 {
+			Condition {
+				Type ControlExists
+				Control "name='Dmic0 Capture Volume'"
+			}
+			True.BootSequence [
+				cset "name='Dmic0 Capture Volume' 70%"
+				cset "name='Dmic0 Capture Switch' on"
+			]
 		}
-		True.BootSequence [
-			cset "name='Dmic0 Capture Volume' 70%"
-			cset "name='Dmic0 Capture Switch' on"
-		]
+		Macro.Dmic0.SofEqBootSetup "endpoint='Dmic0' direction='Capture' eqtype='Iir'"
+		Macro.Dmic0.SofDrcBootSetup "endpoint='Dmic0' direction='Capture'"
+		Macro.Dmic0.SofBeamformerBootSetup "endpoint=Dmic0 direction=Capture"
```

Comment:
Are these three macros should be executed even in case `If.Dmic0` condition is False?

Reply:
I was thinking that in theory the pipeline could be be without volume control but with these processing components. Now it's enough that there is a DeviceDmic defined to make the checks for these controls. Though we do have always with PCH-DMIC the DSP volume control so these could be in above True {}. Maybe I should move.

Good that you asked --  I realized now that I should have these three macro lines added also to sof-soundwire.conf. There are PCs with soundwire headset/speaker codec but with PCH-DMIC.